touchstone/lib/managers/docker_manager.py

```diff
@@ -94,14 +94,18 @@ def __build_env_str(self, environment_vars: List[Tuple[str, str]] = []) -> str:
             additional_params += f' -e {var}="{value}"'
         return additional_params[1:]
 
-    def __run_image(self, additional_params: str, image: str, hostname: str = None) -> str:
+    def __run_image(self, additional_params: str, image: str, log_path: Optional[str], hostname: str = None) -> str:
```

Comment:
The use of `Optional` is patterned after `__execute_image` (line 111/115, below) but requires the parameter to be before `hostname`, meaning the `__run_image` call (line 52, above) has to add `log_path` as the second-last parameter rather than the last. We could instead use `log_path: str = None` and make it the last parameter. I am not deeply familiar with `Optional` in Python, couldn't say why `__execute_image` used it.

Reply:
I believe the use of Optional here was to convey that log_path is a required param, but could be None if logging is disabled